examples/analytics.amp.html

```diff
@@ -218,6 +218,25 @@
 </script>
 </amp-analytics>
 
+<!-- Lotame tracking.
+  Please replace the value for "account" (12345)
+  with the correct Lotame Client ID.
+-->
+<amp-analytics type="lotame">
+<script type="application/json">
+{
+  "vars": {
+    "account": 12345
+  },
+  "extraUrlParams": {
```

Comment:
The extraUrlParams were added as an example for our clients. They were left intentionally generic to illustrate that they could be anything and that the specifics are dependent on case. Would you prefer I remove them?